.github/workflows/delete-packagecloud-packages.yml

```
@@ -0,0 +1,37 @@
+name: PackageCloud nightly package delete schedule
+
+env:
+  PACKAGE_CLOUD_API_TOKEN: ${{ secrets.PACKAGE_CLOUD_API_TOKEN }}
+
+on:
+  push:
+    branches:
+      - "**"
+    schedule:
+      - cron: "0 18 * * *"
```

I do not think that on.push.schedule is valid.

```suggestion
on:
  schedule:
    - cron: '0 18 * * *'

```

---

I added it just to execute for tests. Because executing it when opening new PR it is useful to make sure it isn't broken. If you want I can remove by the way

---

It makes sense to keep `on.push.branches` during development then. You can remove that before merges.

However, `on.push.schedule` is not valid and you have an indentation problem here.

See https://github.com/citusdata/tools/pull/214/files#r794111658 for my suggested fix

---

Fixed as you suggested